srcpkgs/m4ri/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'm4ri'
+pkgname=m4ri
+version=20200125
+revision=1
+build_style=gnu-configure
+configure_args="--enable-openmp"
+hostmakedepends=pkg-config
+makedepends=libpng-devel
+short_desc="Library for fast arithmetic with dense matrices over GF(2)"
+maintainer="Eloi Torrents <eloitorrentsj@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://bitbucket.org/malb/m4ri"
+distfiles="https://bitbucket.org/malb/m4ri/downloads/${pkgname}-${version}.tar.gz"
+checksum=0dfb34aed351882a0f2281535ea6f81c690a5efeb14edab131d9ba0dffe44863
+
+m4ri-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/*.a
```

Comment:
This one needs quotes. Whenever there's globbing (* or ?) you need quotes.

For the others, quotes are optional, and the team prefers you to NOT use quotes when they are not necessary.